### DISCUSSION TITLE
Latex tabel in MARP

### DISCUSSION THREAD

**Discussion by fredzett** [marp-team/marp#205] [Q&A] [answered]
Is it possible to use Latex tables in MARP?

I can use `$` (inline) and `$$` to use latex formulas. However, when I do the following it doesn't work. 

```
$$
\begin{table}[]
\begin{tabular}{|l|l|l|}
\hline
A & B & C \\ \hline
1 & 2 & 3 \\ \hline
4 & 5 & 6 \\ \hline
\end{tabular}
\end{table}
$$
```

I am using mathjax. 

Of course I can do Markdown tables. But I was just wondering why this does not work?

Thanks !

**Comment by yhatt** [comment, marked as answer]
> Because `tabular` is not a Math environment? Why should it be part of MathJax?
>
> _-- https://math.meta.stackexchange.com/questions/2016/tabular-in-mathjax_

Instead you can use `array` environment.
https://stackoverflow.com/questions/51803244/how-to-add-table-in-web-page-using-mathjax

**Reply by fredzett** [reply]
Thanks for your swift reply! 

...and thanks for MARP!
